### ISSUE TITLE
Facing problem in training pascal voc model

### ISSUE THREAD

**Issue by mirraaj** [DavideA/dilation-keras#2]
I am facing problem in training my dataset of pascal voc(smaller one) on this project. How to implement category crossentropy for pixel level segmentation.

**Comment by DavideA** [comment]
Hi!
You can easily impement categorical crossentropy in keras code. Assuming your groundtruth and predictions are shaped (batchsize, channels, height, width):
`cce = K.mean(y_true * K.log(y_pred + K.epsilon()), axis=[1, 2, 3])`

Alternatively, you can use the one [already implemented in keras](https://keras.io/losses/).

Hope this helps, 
D
